pkg/kinflate/util/util.go

```diff
@@ -124,6 +117,21 @@ func Encode(in types.KObject) ([]byte, error) {
 	return buf.Bytes(), nil
 }
 
+// EncodeFromKObject encodes the map `in` and output the encoded objects separated by `---`.
+func EncodeFromKObject(in types.KObject) ([]byte, error) {
```

Comment:
Should we call it EncodeKObject(....) ?

We can also implement `Encode` method on types.KObject type, will be better I guess and eventually it will move to `ResourceCollection` ?

Reply:
We can call it `Encode` and move it to package of `ResourceCollection`. We'll leave the `Encode(objs []*unstructured.Unstructured) ([]byte, error)` in utils here.